docs/components/IframeConfigurator/index.tsx

```
@@ -351,6 +358,52 @@ export default function IframeConfigurator() {
+                control={form.control}
+                name="config.channelId"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>Channel ID</FormLabel>
+                    <FormDescription>
+                      You can specify a custom channel ID for posting comments.
+                      If you plan to charge users for posting, please ensure
+                      your channel hook follows the{" "}
+                      <a
+                        href="/hooks"
+                        target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)